plugit/routes.py

```diff
@@ -30,21 +30,21 @@ def mail():
     # (added by the decorator in utils)
     for act in dir(actions):
         obj = getattr(actions, act)
-        if hasattr(obj, 'pi_api_action') and obj.pi_api_action:
+        if hasattr(obj, 'pi_api_action') and obj.pi_api_action and not hasattr(obj.pi_api_action, '__call__'):
```

Comment:
Yes, because `pi_api_route` was missing from those objects. The condition is here to filter them